api/admin/templates.py

```diff
@@ -4,11 +4,14 @@
 <head>
 <title>Circulation Manager</title>
 <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">
-<link href=\"/admin/static/circulation-web.css\" rel="stylesheet" />
+<link href=\"/admin/static/circulation-web.css\" rel="stylesheet"/>
 </head>
 <body>
+  <p class="error" id="error1" style="color:#1B7FA7;font-family:sans-serif;font-weight:bold;font-size:x-large;margin-top:50px;margin-left:30px"></p>
+  <p class="error" id="error2" style="color:#1B7FA7;font-family:sans-serif;font-size:medium;margin-top:10px;margin-left:30px"></p>
```

Comment:
I mentioned the inline style for this thinking it would be one element. Since there are two elements and most of the rules are the same, you can break it out into a style element:

```
<style>
  .error {
    color: #1B7FA7;
    font-family: sans-serif;
    etc...
  }
</style>
```

Reply:
Done!